build/package/Dockerfile.node16-typescript-toolset

```diff
@@ -1,37 +1,14 @@
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.4
+FROM registry.access.redhat.com/ubi8/nodejs-16
```

Comment:
1. All other image references contain a tag other than `latest`. For this one, it might not make much sense but I'd like to keep it consistent: `registry.access.redhat.com/ubi8/nodejs-16:1`.
2. We need to update the Software Design Specification now and add this base image, plus modify the description of the task/image to point to it (and potentially not mention gcc etc. anymore).